image_preprocessor:
  ros__parameters:
    min_length: 5.0
    min_points: 1
    blur_ksize: 21
    adaptive_blocksize: 11
    adaptive_C: 0.1